docker-compose-distributed.yml

```diff
@@ -0,0 +1,90 @@
+version: '3'
+services:
+  zookeeper:
+    image: confluentinc/cp-zookeeper:5.4.0
+    networks:
+      kafka_distributed_network:
+    environment:
+      ZOOKEEPER_CLIENT_PORT: 2181
+      zk_id: '1'
+    # ports:
+    #  - '2181:2181'
```

Comment:
Shouldn't we remove those commented lines? :thinking:

Reply:
These were on original configuration files so I did not remove them. Users arguably might want to open those ports to outside, so I'd say we can keep them commented here